division by zeroTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 30, in <module>
    1/0
    ~^~
ZeroDivisionError: division by zero
[Errno 2] No such file or directory: 'data.txt'Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 7, in <module>
    with open('data.txt', 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'data.txt'
[Errno 2] No such file or directory: 'data.txt'Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 7, in <module>
    with open('data.txt', 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'data.txt'
name 'path' is not definedTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 35, in <module>
    label1 = tk.Label(root, text= path, fg='blue', font=('helvetica', 12, 'bold'))
                                  ^^^^
NameError: name 'path' is not defined
[Errno 2] No such file or directory: 'data.txt'Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 7, in <module>
    with open('data.txt', 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'data.txt'
[Errno 2] No such file or directory: 'data.txt'Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 7, in <module>
    with open('data.txt', 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'data.txt'
[Errno 2] No such file or directory: 'data.txt'Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 6, in <module>
    with open('data.txt', 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'data.txt'
[Errno 2] No such file or directory: 'data.txt'Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 6, in <module>
    with open('data.txt', 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'data.txt'
name 'top' is not definedTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 41, in <module>
    label = tk.Label(top, text = " FOOD ITEMS")
                     ^^^
NameError: name 'top' is not defined
name 'top' is not definedTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 41, in <module>
    label = tk.Label(top, text = " FOOD ITEMS")
                     ^^^
NameError: name 'top' is not defined
name 'top' is not definedTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 41, in <module>
    label = tk.Label(top, text = " FOOD ITEMS")
                     ^^^
NameError: name 'top' is not defined
magic.replace() missing 1 required positional argument: 'id'Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 35, in <module>
    button1 = tk.Button(text='Replace', command=magic.replace(), bg='brown',fg='white')
                                                ^^^^^^^^^^^^^^^
TypeError: magic.replace() missing 1 required positional argument: 'id'
magic.replace() missing 1 required positional argument: 'id'Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 35, in <module>
    button1 = tk.Button(text='Replace', command=magic.replace(), bg='brown',fg='white')
                                                ^^^^^^^^^^^^^^^
TypeError: magic.replace() missing 1 required positional argument: 'id'
magic.replace() missing 1 required positional argument: 'id'Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 35, in <module>
    button1 = tk.Button(text='Replace', command=magic.replace(), bg='brown',fg='white')
                                                ^^^^^^^^^^^^^^^
TypeError: magic.replace() missing 1 required positional argument: 'id'
name 'label2' is not definedTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 38, in <module>
    button1 = tk.Button(text='Replace', command=magic.replace(entry1.get), bg='brown',fg='white')
                                                ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 13, in replace
    label2.config(text="Updated to {0}".format(id))
    ^^^^^^
NameError: name 'label2' is not defined
name 'label2' is not definedTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 38, in <module>
    button1 = tk.Button(text='Replace', command=magic.replace(entry1.get), bg='brown',fg='white')
                                                ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 13, in replace
    label2.config(text="Updated to {0}".format(id))
    ^^^^^^
NameError: name 'label2' is not defined
name 'label2' is not definedTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 38, in <module>
    button1 = tk.Button(text='Replace', command=magic.replace(), bg='brown',fg='white')
                                                ^^^^^^^^^^^^^^^
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 13, in replace
    label2.config(text="Updated to {0}".format("LOL"))
    ^^^^^^
NameError: name 'label2' is not defined
name 'label2' is not definedTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 38, in <module>
    button1 = tk.Button(text='Replace', command=magic.replace(entry1.get), bg='brown',fg='white')
                                                ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 13, in replace
    label2.config(text="Updated to {0}".format(id))
    ^^^^^^
NameError: name 'label2' is not defined
type object 'magic' has no attribute 'replace'Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 36, in <module>
    button1 = tk.Button(text='Replace', command=magic.replace(entry1.get), bg='brown',fg='white')
                                                ^^^^^^^^^^^^^
AttributeError: type object 'magic' has no attribute 'replace'
name 'openpage' is not definedTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 37, in <module>
    button4 = tk.Button(text='Poke with Forms List', command=openpage, bg='brown',fg='white')
                                                             ^^^^^^^^
NameError: name 'openpage' is not defined
name 'label2' is not definedTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 38, in <module>
    button1 = tk.Button(text='Replace', command=magic.replace(entry1.get), bg='brown',fg='white')
                                                ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 13, in replace
    label2.config(text="Updated to {0}".format(id))
    ^^^^^^
NameError: name 'label2' is not defined
name 'label2' is not definedTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 38, in <module>
    button1 = tk.Button(text='Replace', command=magic.replace(entry1.get), bg='brown',fg='white')
                                                ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 13, in replace
    label2.config(text="Updated to {0}".format(id))
    ^^^^^^
NameError: name 'label2' is not defined
module 'tkinter' has no attribute 'ImageTk'Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 54, in <module>
    img = tk.ImageTk.PhotoImage(tk.Image.open("rotom.png"))
          ^^^^^^^^^^
AttributeError: module 'tkinter' has no attribute 'ImageTk'
module 'tkinter' has no attribute 'ImageTk'Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 54, in <module>
    img = tk.ImageTk.PhotoImage(tk.Image.open("rotom.png"))
          ^^^^^^^^^^
AttributeError: module 'tkinter' has no attribute 'ImageTk'
module 'tkinter' has no attribute 'ImageTk'Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 54, in <module>
    img = tk.ImageTk.PhotoImage(tk.Image.open("rotom.png"))
          ^^^^^^^^^^
AttributeError: module 'tkinter' has no attribute 'ImageTk'
module 'tkinter' has no attribute 'ImageTk'Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 54, in <module>
    img = tk.ImageTk.PhotoImage(tk.Image.open("./rotom.png"))
          ^^^^^^^^^^
AttributeError: module 'tkinter' has no attribute 'ImageTk'
module 'tkinter' has no attribute 'ImageTk'Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 55, in <module>
    img = tk.ImageTk.PhotoImage(tk.Image.open("./rotom.png"))
          ^^^^^^^^^^
AttributeError: module 'tkinter' has no attribute 'ImageTk'
name 'img' is not definedTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 57, in <module>
    label4 = tk.Label(frame, image = img)
                                     ^^^
NameError: name 'img' is not defined
module 'tkinter' has no attribute 'ImageTk'Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview\main.py", line 53, in <module>
    img = tk.ImageTk.PhotoImage(tk.Image.open("./rotom.png"))
          ^^^^^^^^^^
AttributeError: module 'tkinter' has no attribute 'ImageTk'
name 'canvas1' is not definedTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview - Copy\main.py", line 52, in <module>
    canvas1.create_window(330,50,window=entry1)
    ^^^^^^^
NameError: name 'canvas1' is not defined
Grid.grid_configure() takes from 1 to 2 positional arguments but 3 were givenTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview - Copy\main.py", line 57, in <module>
    labelx.grid(50,50)
TypeError: Grid.grid_configure() takes from 1 to 2 positional arguments but 3 were given
name 'canvas1' is not definedTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview - Copy\main.py", line 77, in <module>
    canvas1.create_window(330,30,window=EnterDexLabel)
    ^^^^^^^
NameError: name 'canvas1' is not defined
unknown option "-padx"Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview - Copy\main.py", line 76, in <module>
    EnterDexLabel.place(padx=5,pady=5,relx=0,rely=0.1)
  File "C:\Users\Avi\AppData\Local\Programs\Python\Python311\Lib\tkinter\__init__.py", line 2504, in place_configure
    self.tk.call(
_tkinter.TclError: unknown option "-padx"
cannot use geometry manager grid inside . which already has slaves managed by packTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview - Copy\main.py", line 88, in <module>
    GFXReplaceButton.grid(row=2,column=0,padx=10,pady=10)
  File "C:\Users\Avi\AppData\Local\Programs\Python\Python311\Lib\tkinter\__init__.py", line 2549, in grid_configure
    self.tk.call(
_tkinter.TclError: cannot use geometry manager grid inside . which already has slaves managed by pack
cannot use geometry manager grid inside . which already has slaves managed by packTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview - Copy\main.py", line 88, in <module>
    GFXReplaceButton.grid(row=2,column=0)
  File "C:\Users\Avi\AppData\Local\Programs\Python\Python311\Lib\tkinter\__init__.py", line 2549, in grid_configure
    self.tk.call(
_tkinter.TclError: cannot use geometry manager grid inside . which already has slaves managed by pack
cannot use geometry manager grid inside . which already has slaves managed by packTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview - Copy\main.py", line 88, in <module>
    GFXReplaceButton.grid(row=2,column=0)
  File "C:\Users\Avi\AppData\Local\Programs\Python\Python311\Lib\tkinter\__init__.py", line 2549, in grid_configure
    self.tk.call(
_tkinter.TclError: cannot use geometry manager grid inside . which already has slaves managed by pack
cannot use geometry manager grid inside . which already has slaves managed by packTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview - Copy\main.py", line 88, in <module>
    GFXReplaceButton.grid(row=2,column=0)
  File "C:\Users\Avi\AppData\Local\Programs\Python\Python311\Lib\tkinter\__init__.py", line 2549, in grid_configure
    self.tk.call(
_tkinter.TclError: cannot use geometry manager grid inside . which already has slaves managed by pack
unknown option "-row"Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview - Copy\main.py", line 87, in <module>
    GFXReplaceButton.place(row=2,column=0)
  File "C:\Users\Avi\AppData\Local\Programs\Python\Python311\Lib\tkinter\__init__.py", line 2504, in place_configure
    self.tk.call(
_tkinter.TclError: unknown option "-row"
unknown option "-row"Traceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview - Copy\main.py", line 87, in <module>
    GFXReplaceButton.place(row=3,column=0)
  File "C:\Users\Avi\AppData\Local\Programs\Python\Python311\Lib\tkinter\__init__.py", line 2504, in place_configure
    self.tk.call(
_tkinter.TclError: unknown option "-row"
cannot use geometry manager pack inside .!notebook.!frame2 which already has slaves managed by gridTraceback (most recent call last):
  File "C:\Users\Avi\Desktop\rotoview - Copy\main.py", line 87, in <module>
    GFXReplaceButton.pack()
  File "C:\Users\Avi\AppData\Local\Programs\Python\Python311\Lib\tkinter\__init__.py", line 2452, in pack_configure
    self.tk.call(
_tkinter.TclError: cannot use geometry manager pack inside .!notebook.!frame2 which already has slaves managed by grid
